airbyte-integrations/connectors/source-mixpanel/setup.py

```diff
@@ -6,7 +6,7 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk~=0.2",
+    "airbyte-cdk==0.67",
```

Comment:
why?

Reply:
@debanjan97 The format of state events has changed and it isn't the same as we expect in rudder sources.
We need to fix rudder sources to parse the new schema of the state event.